Code-the-Dream-School/bb-practicum-team-3-front
# Oksana hotel details

## Description

Added Hotel Details Page that includes image gallery, available rooms, reviews, amenities and more. In the room section, the user is able to update checkin, checkout, guest number and it will reload and show new available rooms. In the room section as well the user is able to pick number of rooms and see the reservation summary, which includes checkin, checkout dates, number of nights and checkout price. 



## Related Issue



## Acceptance Criteria



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before




### After

![image](https://user-images.githubusercontent.com/84487850/236767638-82625410-54c8-4e9e-8f62-87587ff51d87.png)
![image](https://user-images.githubusercontent.com/84487850/236767751-26cec609-f681-4b72-b06f-bf8b22e88218.png)
![image](https://user-images.githubusercontent.com/84487850/236767817-d1b83134-4d05-4db1-94b1-018684469185.png)
![image](https://user-images.githubusercontent.com/84487850/236767971-9e76d34e-1cde-487a-9eca-2a137b491331.png)



## Testing Steps / QA Criteria

In the Room Search Form pick the chekin date, checkout date and number of guests and click Update Rooms. The page should update the room options. 

In the Add Room dropdown, chose the number of rooms. The Reservation Details box on the right should update the info and price accordingly.
